### PR TITLE
Mbarberi

### PR DESCRIPTION
**Pull request by foobarberis**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,7 +22,7 @@ SRCS		:=	main.cpp \
 				Server/Server.cpp \
 				Server/Quit.cpp \
 				Server/User.cpp \
-				addToQueu.cpp \
```
Comment (VBurton97) [inline Makefile:25]: Why is it deleted?

Comment (VBurton97) [reply Makefile:25]: I hadn't seen that addToQueu ad moove into utils

**Review by VBurton97 — COMMENTED** [review]
May be we could have insert "addToQueu" into one of our already existing file insted of creating a new one.
Also I have already fix the macro ERR_NEEDMOREPARAMS in all files in my branch.

File: src/Server/Server.cpp [hunk 2]
```diff
@@ -69,17 +56,20 @@ bool Server::processMessage( message_t message, std::deque<message_t> & msgQueue
...
     }
     std::string word = message.msg.substr( 0, posFirstSpace );
-    if ( this->_clients.count( message.fd ) == 0 && word != "PASS" ) {
+    if ( word == "CAP" ) {
+        /* Drop any CAP message */
+        return 0;
+    } else if ( ( this->_clients.count( message.fd ) == 0 ) && ( word != "PASS" ) ) {
         /* Drop all messages if the client has not registered properly */
-        return false;
+        return -1;
```
Comment (touline-p) [inline src/Server/Server.cpp:67]: Peut etre que des define serait plus lisible

**Review by touline-p — APPROVED** [review]
rly goot arh arh

**Pull request merged**